pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
+norecursedirs = .tox
```

Comment:
dunno if it makes much of a noticeable difference to speed of test run in the grand scheme of things, but there's some other stuff like `/locale` and `/static` that we can usefully exclude here